core/src/api/TezosConfiguration.cpp

```diff
@@ -7,4 +7,6 @@ namespace ledger { namespace core { namespace api {
 
 std::string const TezosConfiguration::TEZOS_XPUB_CURVE = {"TEZOS_XPUB_CURVE"};
 
+std::string const TezosConfiguration::TEZOS_PROTOCOL_UPDATE = {"TEZOS_PROTOCOL_UPDATE"};
```

Comment:
do we need to send this constants or it's set as default?

Reply:
For the moment, the default is the old protocol, so you need to inject that to the wallet's configuration, set `TEZOS_PROTOCOL_UPDATE ` to `TEZOS_PROTOCOL_UPDATE_BABYLON`

Reply:
cc @valpinkman . derivation.js can set custom param per derivation. maybe need same concept somewhere **per crypto**